pkg/grpc/middleware/authz/authorizer.go

```diff
@@ -81,7 +81,7 @@ func (a *Authorizer) authorize(ctx context.Context, method string) (context.Cont
 		return ctx, nil
 	}
 
-	if a.DontEnforce {
+	if a.Enforce {
```

Comment:
```suggestion
	if !a.Enforce {
```

that seems crucial?

Reply:
🤦 Was debugging unrelated issue and changed that for a quick test…